pyproject.toml

```diff
@@ -97,7 +98,10 @@ warn_unused_ignores = true
 namespace_packages = true
 show_error_codes = true
 show_error_context = true
-plugins = ["mypy_zope:plugin"]
+plugins = [
+    "mypy_zope:plugin",
+    "pydantic.mypy"
```

Comment:
Fix this, the plugin is not working

Reply:
For some reason it was conflicting with `mypy_zope:plugin`. I found out that inverting the order works.